backend/GymBeam/Controllers/UserController.cs

```diff
@@ -6,6 +6,8 @@
 using GymBeam.Constants;
 using GymBeam.Queries;
 using MediatR;
+using System.Net;
```

Comment:
Do we need this additional refs?

Reply:
It is necessary for HttpStatusCode (status code returned when an exception is caught)